package.json

```diff
@@ -24,7 +25,9 @@
     "build": "gatsby build",
     "serve": "gatsby serve",
     "clean": "gatsby clean",
-    "test:links": "remark src/pages --ignore-pattern src/pages/graphql,src/pages/_includes --quiet --frail"
+    "test:links": "remark src/pages --quiet --frail",
```

Comment:
If you remove the ignore pattern, the link checker will fail on false positives. See the description in https://github.com/AdobeDocs/commerce-webapi/pull/57.